android/res/values/strings.xml

```diff
@@ -1998,4 +1998,5 @@
 	<string name="type.piste_type.sled">Sledding Piste</string>
 	<string name="type.building_part">Building</string>
 	<string name="type.amenity.events_venue">Events Venue</string>
+	<string name="comma_separated_pair">%1$s, %2$s</string>
```

Comment:
This string should be added into data/strings/strings.txt, and then regenerated using tools/unix/generate_localizations.sh

Reply:
The commit with the result of the regeneration (this changed line and any others) should be a separate one, with the title "[strings] Regenerated"